core/src/main/java/uk/ac/bbsrc/tgac/miso/core/data/type/QcType.java

```diff
@@ -257,4 +259,20 @@ public long getId() {
   public String getAlias() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getName" on line 93. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)